src/VirtualMonoRepo/source-mappings.json

```diff
@@ -160,7 +160,11 @@
                 "src/application-insights/**/NuGet.config",
                 "src/humanizer/samples/**/*.js",
                 "src/newtonsoft-json/**/NuGet.Config",
+                "src/spectre-console/docs/input/assets/CascadiaMonoPL.woff2",
+                "src/spectre-console/docs/input/assets/images/table.mp4",
+                "src/spectre-console/docs/input/assets/images/table.webm",
                 "src/spectre-console/docs/package.json",
+                "src/spectre-console/docs/src/SocialCards/CascadiaCodePL.woff2",
```

Comment:
I'm wondering whether we should just cloak the entire `docs` directory. Thoughts on that @MichaelSimons?

Reply:
I lean towards cloaking the entire documentation directory.